### PR TITLE
5702 - Add readme

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,51 @@
...
+
+`$ hn-happ-create my-project-name`
+
+This will create a new directory for your project, and start a holochain conductor and the UI server.
+
+The first time the holochain conductor runs there may be some additional compilation so it might take a little while.
+
+Once it's complete, a browser page should open to the default notes happ.
+
+If it doesn't, you can browse to localhost:3000 to use the happ.
```
Comment (pdaoust) [inline README.md]:
```suggestion
If it doesn't, you can browse to localhost:5200 to use the happ.
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,51 @@
...
+
```
Comment (pdaoust) [inline README.md:61]:
```suggestion

The conductor and the UI server run in the foreground, so you can stop them by pressing `Ctrl`+`C`. If you want to restart them, run this command in the project's directory (make sure you're still in the nix shell):

`$ npm start`

```

**Review by pdaoust — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -1,4 +1,51 @@
...
-curl -L -o master.tar.gz https://github.com/holochain/react-graphql-template/archive/master.tar.gz
-mkdir happ-template
-tar -zxvf master.tar.gz --strip-components=1 -C ./happ-template
+The `hn-happ-create` script is a simple dev tool that allows you to get a holochain development environment up and running as fast as possible.
+
+The script creates a simple "notes happ" that allows you to create, edit, delete and list notes. It includes a dna backend and a react + graphql frontend. It is a minimal working Holochain app.
+
+## 0 - nix-shell
+
+Install nix-shell on your machine. Here are our instructions on how to do that [nix-shell installation](fixme)
```
Comment (pdaoust) [inline README.md]:
```suggestion
Install nix-shell on your machine, using this one-line command:

`curl https://nixos.org/nix/install | sh`

(Note: we currently support macOS and Linux only; please see our [development environment setup guide](https://developer.holochain.org/docs/install/) to set up Linux and nix-shell on Windows.)
```

**Comment by pdaoust** [comment]
@robbiecarlton I added a couple things in 0c97972241376d38f8e2fcd3b4dea0c61c842cc0 and 8772c2f277272a098dded28ab2156e58fd17cccb -- could you let me know what you think of them? (do they contribute something meaningful? are they accurate?)

If it all looks good, I think the readme is ready to merge!

File: README.md [hunk 4]
```diff
@@ -1,4 +1,88 @@
...
+## A very brief tour
+
+The UI code is in `ui_src` and the DNA code is in `dna_src`. This notes demo app uses:
+
+1. A Holochain DNA on the back end (of course)
+2. [Apollo GraphQL middleware](https://www.apollographql.com/) in the browser to translate zome calls into something easy for front-end frameworks to understand and manipulate
+3. [React](https://reactjs.org) for UI and data flow
+4. [create-react-app](https://create-react-app.dev/) for development tooling
+
+As with most hApps, much of the business logic lives in the front-end and is delivered as static assets to the browser. The create-react-app dev tooling builds it all from source, gets it ready for the browser, and serves it up using [Webpack's dev server](https://webpack.js.org/configuration/dev-server/). (We're using this instead of Holochain's built-in static asset server because it supports live reloading of changes to the UI code.)
```
Comment (robbiecarlton) [inline README.md:67]: @pdaoust good summary!

**Comment by pdaoust** [comment]
@robbiecarlton this looks like it's ready to go; should we merge it?

**Pull request merged**